### PR TITLE
TrustedSigning: CLI extension commands

### PR DESCRIPTION
**Pull request by amritas**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [X] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [X] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [X] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [X] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

**Comment by amritas** [comment]
_azdev style_ and _linter_ are working locally, but failing here. Please suggest what needs to be done.

File: src/trustedsigning/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Azure CLI Trustedsigning Extension #
+This is an extension to Azure CLI to manage Trustedsigning resources.
+
+## How to use ##
+Please add commands usage here.
```
Comment (necusjz) [inline src/trustedsigning/README.md:5]: please complete this part

Comment (amritas) [reply src/trustedsigning/README.md]: Updated

**Comment by amritas** [comment]
@necusjz Please merge the PR

**Review by necusjz — APPROVED** [review]

**Pull request merged**